chapters/hpmor-chapter-025.tex

```diff
@@ -1,7 +1,13 @@
 \chapter{Hold Off on Proposing Solutions}
 
-\section{Act 2:\protect\footnotemark}
-\authorsnotetext{Since the science in this story is usually all correct, I include a  warning that in Chapters~22–25 Harry overlooks many possibilities, the most  important of which is that there are lots of magical genes but they're all on  one chromosome (which wouldn't happen naturally, but the chromosome might have  been engineered). In this case, the inheritance pattern would be Mendelian, but  the magical chromosome could still be degraded by chromosomal crossover with  its non-magical homologue. (Harry has read about Mendel and chromosomes in  science history books, but he hasn't studied enough actual genetics to know  about chromosomal crossover. Hey, he's only eleven.) However, although a modern  science journal would find a \emph{lot} more nits to pick, everything Harry  presents as strong evidence is in fact strong evidence---the other  possibilities are \emph{improbable}.}
+\begin{chapterOpeningAuthorNote}
+Since the science in this story is usually all correct, I include a warning that in Chapters~22–25 Harry overlooks many possibilities, the most important of which is that there are lots of magical genes but they're all on one chromosome (which wouldn't happen naturally, but the chromosome might have been engineered). In this case, the inheritance pattern would be Mendelian, but the magical chromosome could still be degraded by chromosomal crossover with its non-magical homologue. (Harry has read about Mendel and chromosomes in science history books, but he hasn't studied enough actual genetics to know about chromosomal crossover. Hey, he's only eleven.) However, although a modern science journal would find a \emph{lot} more nits to pick, everything Harry presents as strong evidence is in fact strong evidence---the other possibilities are \emph{improbable}.
```

Comment:
replace "---" with "—" (is this in fact necessary for ePUB conversion?). Anyway, good for consistency.

Reply:
good point, also found this in chapter 25